src/com/cdd/bao/template/Vocabulary.java

```diff
@@ -401,13 +415,19 @@ private void loadLabels(File baseDir, File[] extra, Set<String> exclude) throws
 			Statement stmt = iter.next();
 			Resource subject = stmt.getSubject();
 			Resource object = stmt.getObject().asResource();
-			classBreakers.add(subject.getURI() + "::" + object.getURI());
+
+			String subjURI = remapIfAny(subject.getURI());
+			String objURI = remapIfAny(object.getURI());
+			classBreakers.add(subjURI + "::" + objURI);
 		}
 		for (StmtIterator iter = model.listStatements(null, rdfType, resEliminated); iter.hasNext();)
 		{
 			Statement stmt = iter.next();
-			eliminatedTerms.add(stmt.getSubject().getURI());
+			List<String> eliminatedURIs = remapSequenceIfAny(stmt.getSubject().getURI());
```

Comment:
Disregard original comment - it's correct, but redundant. Iterating over the keySet accomplishes the goal.

Reply:
I see your point.  Let me make that change.